### ISSUE TITLE
Url's with "[" cannot be loaded

### ISSUE THREAD

**Issue by subutux** [Freso/script.module.urllib3#2]
Hi, 
I don't know if this is the right place for it, but it seems as of today, there was a new update (1.25.1), breaking some urls. I'm using one plugin that requests urls containing "[" and "]" in them, resulting the following execption thrown:
```
InvalidURL: Failed to parse: https://vrtnu-api.vrt.be/suggest?facets[categories]=met-audiodescriptie
```
of course, the url is correct.

**Comment by subutux** [comment]
ok wrong repo.. https://github.com/urllib3/urllib3/issues/1585

**Comment by Freso** [comment]
I’ve subscribed to that issue now. I’ll try and get an update of `urllib3` into Kodi’s repositories as soon as they publish a fix. Seems like you’re not the only affected person, so hopefully it won’t take too long!

Thanks for letting me know, even if it’s ultimately an upstream issue! :)

**Comment by Freso** [comment]
Upstream has released 1.25.2 which should have a fix for this issue, and I’ve requested an update for this add‐on into the Kodi repository: https://github.com/xbmc/repo-scripts/pull/1052
